docs/source/developer_zone/changes-2.0.rst

```diff
@@ -0,0 +1,90 @@
+=================
+Mamba 2.0 Changes
+=================
+.. ...................... ..
+.. THIS IS STILL A DRAFT ..
+.. ...................... ..
+
+.. TODO high-level summary of new features:
+.. - OCI registries
```

Comment:
🤯 As someone who actually does use `powerloader` to publish `conda` packages to `ghcr.io` I'm super excited to have this functionality in `mamba` directly! :heart::rocket: